src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -2381,6 +2382,17 @@ public function setIncrementing($value)
     }
 
     /**
+     * Transform each attribute in the model using a callback.
+     *
+     * @param  callable  $callback
+     * @return \Illuminate\Support\Fluent
+     */
+    public function transform(callable $callback)
+    {
+        return (new Fluent($this->toArray()))->transform($callback);
```

Comment:
Yes, I got the idea and it is good. Although I think that some `Transformer` contract would be useful.